README.md

```diff
@@ -42,3 +42,27 @@ It contains two main modes:
 [Learn more](./crox/README.md)
 
 [wg-self-profile]: https://rust-lang.github.io/compiler-team/working-groups/self-profile/
+
+### mmedit
+
+`mmedit` is for editing `.mm_profdata` files generated by `measureme`.
+
+[Learn more](./mmedit/README.md)
+
+### mmview
+
+`mmview` is for printing the event data generated by `measureme`.
+
+[Learn more](./mmview/README.md)
+
+### analyzeme
+
+`mmview` is for printing the event data generated by `measureme`.
```

Comment:
`analyzeme` is a library with common functionality for measureme tools

Reply:
Updated. Thanks!